README.md

```diff
@@ -1 +1,72 @@
+46. What is Object Destructuring? [Answer](https://youtu.be/dTZk9yv2Bp0)
+47. What are ES6 Modules? [Answer](https://youtu.be/CkXs9R9JMbA)
+48. What is the Set object and how does it work? [Answer](https://youtu.be/0rHYrnDCMsc)
+49.  What is a Callback function? [Answer](https://youtu.be/8KVT2aViRdQ)
+50.  What are Promises? [Answer](https://youtu.be/OUeHvsFBT2E)
+51.  What´s the diferrence between Spread Operator and Rest Operator? [Answer](https://youtu.be/uunZfKjPfeE)
+52.  What is async/await and How does it work? [Answer](https://youtu.be/ziUiQZcMv3g)
+53.  what are default parameters? [Answer](https://youtu.be/RsDTr5HZjck)
+54.  What are wrapper objects? [Answer](https://youtu.be/nl-mv3d823w)
+55.  What is the difference betweem Implicit and Explicit Coercion? [Answer](https://youtu.be/PVFCx8YOFKo)
+56.  What is NaN? and How to check if a value is NaN? [Answer](https://youtu.be/OF8dTAkA4BE)
+57.  How to check if a value is an Array? [Answer](https://youtu.be/yMuLw6j2TEE)
+58.  How to check if a number is even without using the % or modulo operator? [Answer](https://drive.google.com/file/d/1VIv6G5JOXS6uD4SSianBKgyXg6X62PX2/view?usp=drive_link)
+59.  How to check if a certain property exists in an object? [Answer](https://drive.google.com/file/d/15y_sGt5QPAFHudwWEEhzEhINrP2CEXON/view?usp=drive_link)
+60.  What is AJAX? [Answer](https://youtu.be/HKWuMyLj6Rw)
+61.  What are the ways of making objects in JavaScript? [Answer](https://drive.google.com/file/d/1X16lMguCR0cOpGNKqmEwB73RMm67Ruk7/view?usp=drive_link)
+62.  What´s the difference between Object.seal and Object.freeze methods? [Answer](https://drive.google.com/file/d/1HXWox9vj6yJqPvKbhtcrdwmND_PNFgAE/view?usp=drive_link)
+63.  what´s the difference between the in operator and and the hasOwnProperty method in objects? [Answer](https://drive.google.com/file/d/11rsTAIClRVNjiwSPwi4pIHNOJYFn1ZVq/view?usp=drive_link)
+64.  What are the ways to deal with Asynchronous Code in JavaScript? [Answer](https://drive.google.com/file/d/1lFA_KqXy8SIa9Ph1f2r4hbBmVBO0ZYZJ/view?usp=drive_link)
+65.  What´s the difference between a function expression and functions declaration? [Answer](https://drive.google.com/file/d/13RAX3zHdYGsfxgaSAoMdldNc7CeXynU5/view?usp=sharing)
+66.  How many ways can function be invoked? [Answer](https://drive.google.com/file/d/1QuRnDDFOQaQIu3DBrEfiHUSiWNN16e6w/view?usp=sharing)
+67.  What is memoization and what´s the use it? [Answer](https://drive.google.com/file/d/1Fxjj8-sHdMmINHQjNFyCMUau-8eAm49a/view?usp=sharing)
+68.  Implement a memoization helper function [Answer](https://drive.google.com/file/d/1IoKviCljf2afx2snEq-A6Xyg8ZODBKgq/view?usp=sharing)
+69.  Why does typeof null return object? How to check if a value is null? [Answer](https://drive.google.com/file/d/1m_oFTY9WTddFQlyTDc7r50zkUvhHQnZv/view?usp=sharing)
+70.  What does the new keyword do? [Answer](https://drive.google.com/file/d/1JYMaQDAClgk9v_gVu2QrtkQnPqeq4XDM/view?usp=sharing)
```

Comment:
**Revisión de respuestas 61-70 - Bloque G**
Ma parecieron bastante completas las preguntas, excelente trabajo.
Como observaciones me gustaría comentar lo siguiente:
Pregunta 70.- ¿Qué hace el opreador new? - Sería interesante comentar que el operador new no retorna tipos primivitos. Es posible retornar valores no primitivos que no correspondan a this(la instancia del objeto) cuando retornamos algún valolr explícitamente. 
Algo como 
```JavaScript
function Thing() {
  this.one = 1;
  this.two = 2;

  return {
    three: 3,
    four: 4
  };
}

var myThing = new Thing();
console.log(myThing) // {three:3, four:4} not this on Thing constructor

```